ListaDeTarefas/ViewModel/TelaInicialTableViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  TelaInicialTableViewController.swift
+//  ListaDeTarefas
+//
+//  Created by Angélica Andrade de Meira on 25/04/22.
+//
+
+import UIKit
+import CoreData
+
+class TelaInicialTableViewController: UITableViewController {
+    
+    var contexto: NSManagedObjectContext?
+    var listaDeListas: [NSManagedObject]? = []
+    var listaSelecionada: NSManagedObject?
+    
+    // MARK: - View code
+    
+    private lazy var botaoAdicionarLista: UIBarButtonItem = {
+        let view = UIBarButtonItem()
+        view.title = "Nova Lista"
+        view.target = self
+        view.action = #selector(vaiParaAdicionarLista)
+        return view
+    }()
+    
+    @objc func vaiParaAdicionarLista() {
+        self.present(UINavigationController(rootViewController: AdicionaListaViewController()), animated: true)
+    }
+    
+    override func loadView() {
+        super.loadView()
+        self.navigationItem.title = "Minhas listas"
+        self.navigationItem.rightBarButtonItems = [botaoAdicionarLista]
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.tableView.register(UITableViewCell.self, forCellReuseIdentifier: "celulaMinhasListas")
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
+        contexto = appDelegate.persistentContainer.viewContext
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        recuperaListas()
+    }
+    
+    func recuperaListas() {
+        let requisicao = NSFetchRequest<NSFetchRequestResult>(entityName: "Lista")
+        let ordenacao = NSSortDescriptor(key: "descricao", ascending: true)
+        requisicao.sortDescriptors = [ordenacao]
+        
+        do {
+            
```

Comment:
não coloque linhas em branco desnecessárias entre o início de blocos e seu conteúdo